tests/BasicRegistryTests.cpp

```diff
@@ -2633,4 +2777,285 @@ TEST_CASE("BasicRegistryTests::multi-strings", "[registry][get_registry_string]"
 #endif
 #endif
 }
+#endif
+
+#if defined(_VECTOR_) && defined(WIL_ENABLE_EXCEPTIONS)
+TEST_CASE("BasicRegistryTests::vector-bytes", "[registry]]")

```

Comment:
Do we have any tests where we write a REG_SZ (perhaps with embedded nulls) and then read it with `get_value_byte_vector`?

Reply:
yep. that test case was added.

Reply:
I may have removed it accidentally, but I only see tests where we *set* using `set_value_byte_vector` and read via a different method. There are a buncha tests where we get & set byte vectors, but none where we *read* a byte vector from a value set via a direct method. We should ensure we *read* strings especially via `get_value_byte_vector` in the tests.